packages/p2p-v2/src/features/my-profile/screens/MyProfileStats/MyProfileStats.scss

```diff
@@ -0,0 +1,68 @@
+.p2p-v2-my-profile-stats {
```

Comment:
can we remove `p2p-v2` from the class name?

Reply:
we need this in order to scope all related classnames under p2p-v2 😅 
If we don't prefix it with `p2p-v2`, later `my-profile-stats` classname from p2p-v2 could override other classnames with the same name `my-profile-stats` 👀

Reply:
I guess removing that will cause duplicate classnames issue

Reply:
Later this classname coming from `p2p` package will be overriden by p2p-v2
![Screenshot 2024-01-08 at 2 12 59 PM](https://github.com/binary-com/deriv-app/assets/103016120/99f84a20-2cae-43ce-831b-7ea85dc72f79)